<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPlainTextEdit" name="GrammerTextEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QTextBrowser" name="displaytext">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="stringTextEdit"/>
          <widget class="QTextBrowser" name="stringTextBrowser"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionCleanFile"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>判断</string>
    </property>
    <addaction name="actioninput"/>
    <addaction name="actionFliestring"/>
    <addaction name="actionRunChar"/>
    <addaction name="actionBack"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="actionRunStart"/>
    <addaction name="actionClean"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>判断任意串</string>
   </property>
  </action>
  <action name="actionNoepsilon">
   <property name="text">
    <string>消除epsilon产生式</string>
   </property>
  </action>
  <action name="actionNounit">
   <property name="text">
    <string>消除单一产生式</string>
   </property>
  </action>
  <action name="actionNouse">
   <property name="text">
    <string>消除无用产生式</string>
   </property>
  </action>
  <action name="actiontoG1">
   <property name="text">
    <string>转换G1文法</string>
   </property>
   <property name="iconText">
    <string>转换G1文法</string>
   </property>
  </action>
  <action name="actiontoG2">
   <property name="text">
    <string>转换G2文法</string>
   </property>
  </action>
  <action name="actiontoG3">
   <property name="text">
    <string>转换G3文法</string>
   </property>
  </action>
  <action name="action_PDA">
   <property name="text">
    <string>转换PDA</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionepsilon">
   <property name="text">
    <string>消除epsilon产生式</string>
   </property>
  </action>
  <action name="actionunit">
   <property name="text">
    <string>消除单一产生式</string>
   </property>
  </action>
  <action name="actionuse">
   <property name="text">
    <string>消除无用产生式</string>
   </property>
  </action>
  <action name="actionG1">
   <property name="text">
    <string>转成G1文法</string>
   </property>
  </action>
  <action name="actionG2">
   <property name="text">
    <string>转成G2文法</string>
   </property>
  </action>
  <action name="actionG3">
   <property name="text">
    <string>转成G3文法</string>
   </property>
  </action>
  <action name="actionpda">
   <property name="text">
    <string>转成PDA</string>
   </property>
  </action>
  <action name="actioninput">
   <property name="text">
    <string>输入任意串</string>
   </property>
  </action>
  <action name="actionmsg">
   <property name="text">
    <string>msg</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionMsg">
   <property name="text">
    <string>软件信息</string>
   </property>
  </action>
  <action name="actionqt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>查看全部</string>
   </property>
   <property name="iconText">
    <string>运行全部</string>
   </property>
  </action>
  <action name="actionSimple">
   <property name="text">
    <string>CFG化简</string>
   </property>
  </action>
  <action name="actionGNF">
   <property name="text">
    <string>转成GNF</string>
   </property>
  </action>
  <action name="actionPDA">
   <property name="text">
    <string>构造PDA</string>
   </property>
  </action>
  <action name="actionRunChar">
   <property name="text">
    <string>运行判断</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>返回主界面</string>
   </property>
  </action>
  <action name="actionRunStart">
   <property name="text">
    <string>确定运行</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>清除所有</string>
   </property>
  </action>
  <action name="actionCleanFile">
   <property name="text">
    <string>清除文件缓存</string>
   </property>
  </action>
  <action name="actionFliestring">
   <property name="text">
    <string>从文件导入</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
